### ISSUE TITLE
[wavedeform] RELEASE_NOTES needs updating (Trac #1192)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1186]
r134186/IceCube should be added to the release notes.

Migrated from https://code.icecube.wisc.edu/ticket/1192
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "r134186/IceCube should be added to the release notes.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[wavedeform] RELEASE_NOTES needs updating", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-19T16:58:53", 
    "milestone": "", 
    "owner": "jbraun", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150820T13:50:11: 
* jbraun changed resolution from "" to "fixed"
* jbraun changed status from "new" to "closed"
* jbraun commented:

> In changeset 136612:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="136612"
> Add release notes; use unsigned types where needed to appease the static analysis.  closes #1186
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
